src/Data/Quote/QuoteValuesConfig.ts

```diff
@@ -81,8 +81,85 @@ provideDataValues(Quote, {
     'PSA_QUO_STY_SVC',
   ],
 })
+provideAttributeLocalizers('de', Quote, {
+  mainStatus: {
+    PSA_SAP_MAI_QUO_LST: 'Angebot verloren',
+    PSA_SAP_MAI_QUO_ORD_CER: 'Angebotserstellung',
+    PSA_SAP_MAI_QUO_ORD_CRE: 'Angebot gewonnen',
+    PSA_SAP_MAI_QUO_ORD_EPH_VH: 'Verhandlungsphase',
+    PSA_SAP_MAI_QUO_ORD_EPH: 'Entscheidungsphase',
+    PSA_SAP_MAI_QUO_ORD_PRP_ZUR: 'Angebot zurückgestellt',
+  },
+  status: {
+    PSA_PRO_QUO_AS6: 'Angebot OK, Projektabschluss in 3-6 Monaten',
+    PSA_PRO_QUO_CMV: 'Angebot vom Kunden zurückgestellt',
+    PSA_PRO_QUO_COK: 'Angebot gewonnen/Kundenzusage',
+    PSA_PRO_QUO_CPE: 'Angebotsabgabe erfolgt, pending',
```

Comment:
This looks like a strange german translation.

Reply:
According to Ronald these are the localizers from the standard. I don't see any potential todo here.